packages/experimental/Icon/src/Icon.tsx

```diff
@@ -56,16 +56,20 @@ function renderSvg(iconProps: IconProps) {
   const style = mergeStyles(iconProps.style, rasterImageStyleCache({ width: width, height: height }, [width, height])[0]);
 
   // react-native-svg is still on 0.61, and their color prop doesn't handle ColorValue
+  // If a color for the icon is not supplied, fall back to white or black depending on appearance
+  const theme = useTheme();
+  const iconColor = svgIconProps.color ? svgIconProps.color : theme.host.appearance === 'dark' ? '#FFFFFF' : '#000000';
```

Comment:
A nit: We could just have the default value of the color prop be set in Icon itself, and hide the need to set it ourself here.

Reply:
I'm dumb and didn't realize this is Icon.tsx. ignore me!